<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>skinToneDialog</class>
 <widget class="QDialog" name="skinToneDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose Skin Tone</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>70</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>301</width>
     <height>212</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QRadioButton" name="skinColorCheck1">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #874126</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">skinColorButtonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="skinColorCheck2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: #B86A37</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">skinColorButtonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="skinColorCheck3">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: #DE884D</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">skinColorButtonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="skinColorCheck4">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: #EA9F5A</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">skinColorButtonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="skinColorCheck5">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: #F9B972</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">skinColorButtonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="skinColorCheck6">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #FFCB8B</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">skinColorButtonGroup</string>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>skinToneDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>skinToneDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="skinColorButtonGroup"/>
 </buttongroups>
</ui>
